packages/taquito/src/operations/types.ts

```diff
@@ -568,6 +568,16 @@ export interface FailingNoopParams {
   basedOnBlock: BlockIdentifier;
 }
 
+export interface StakingParams {
```

Comment:
You asked yesterday why there is `source?: `in transferParam that it's for people just want to use taquito to prepare the op without any signer or wallet configured, i think it's also needed for `StakingParams` to have optional source